docs/installation.md

```diff
@@ -214,19 +214,16 @@ Want to know more about what the setup scripts are doing?
   snapshot of all of the files neccessary to run a container).
 - **Docker Compose**: Compose allows you to configure and run a collection of
   connected containers (like a web application and its database).
-- **Docker Machine**: In environments where Docker Engine is not available,
-  Docker Machine can be used to create and manage Docker hosts on virtual
-  machines.
 
 ### 1. Setup your Docker environment
 
 If you have never installed Docker before, follow the instructions
-[here](https://docs.docker.com/engine/installation/) or from your operating
-system vendor. If you are on a mac and are unable to install the official
-"Docker for Mac" package, the quickstart instructions below might help.
+[here](https://docs.docker.com/install/) or from your operating system vendor.
+
+The default Docker installation on some systems includes Docker Compose.
+For systems where this is not the case, Docker Compose will need to be
+[installed manually](https://docs.docker.com/compose/install/).
 
-If you are on a machine that is already set up to run Linux docker containers,
-please install [Docker Compose](https://docs.docker.com/compose/install/).
 If `docker-compose ps` runs without error, you can can go to step 2.
```

Comment:
I just did the docker-compose installation today for the very first time using a fresh clone of `cfgov-refresh`, and these changes match the process that worked for me (i.e. realize I had no idea what step 1 was talking about, hope that everything was going to work, and skip right to step 2).

I know this isn't something that changed in this PR, but I did end up copying `.env_SAMPLE` to `.env` even though the instructions told me I could skip to step 2 right before that. I 100% may have messed something up there, but if not and you do indeed still need to copy over `.env_SAMPLE` before proceeding if you're working in a fresh clone, maybe we can rearrange things a bit to make it clear that's still a step you need to do.

Reply:
Great catch, @niqjohnson. I've corrected that in 303c123:

<img width="817" alt="image" src="https://user-images.githubusercontent.com/654645/65550734-3f792f00-deee-11e9-8c4f-56ce63470e07.png">

Reply:
Awesome, that looks great.